### DISCUSSION TITLE
Daybreak

### DISCUSSION THREAD

**Discussion by BobertoBeans** [3kh0/ext-remover#1109] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
In the Kajigs repository, it mentions the exploit "Daybreak" for crosh. I was wondering where I could find it. If you know what it is or how to find it, please help.

**Comment by S-PScripts** [comment]
you can find it in the titanium network discord server (discord.gg/unblock)

if you want it, here it is: (it's by kaitlin)

Method 1
1) Download this extension: https://chrome.google.com/webstore/detail/secure-shell-dev/algkcnfjnajfhgimadimbjhmpaeohhln (or any other SSH extension)

2) Type anything in the username box
3) Type >crosh in the hostname box
4) Press ENTER

You should now have a crosh window open.

Method 2 (requires further testing): 

1) Open the Terminal app and make a new SSH profile

2) Type in ssh a@>crosh

3) Save the profile

4) Right click your terminal app and click on the SSH profile you made

Enjoy your crosh window

If either of these methods don't work, you can try downgrading to any version below v91.
